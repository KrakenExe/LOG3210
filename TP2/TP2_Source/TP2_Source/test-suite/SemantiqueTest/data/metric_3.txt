Declaration
num a;
num b;
listnum [] c;
EndDeclaration

a = - b * 1 ;