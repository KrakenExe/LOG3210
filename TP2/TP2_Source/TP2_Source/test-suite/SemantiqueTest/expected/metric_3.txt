{VAR:3, WHILE:0, IF:0, FOR:0, OP:2}